configs/galasa.json

```diff
@@ -9,10 +9,12 @@
   ],
   "selectors": {
     "lvl0": {
-      "selector": "(//nav[@id='sidebar']//div[contains(concat(' ', normalize-space(@class), ' '), ' markerInNavPath ') and not(contains(concat(' ', normalize-space(@class), ' '), ' selected '))]/a/text() | //nav[@id='headerNav']//a[contains(concat(' ', normalize-space(@class), ' '), ' selected ')]/text())[last()]",
-      "type": "xpath",
+      "selectors": [
+        "#sidebar div.markerInNavPath:not(.selected)",
+        "#headerNav a.selected"
+      ],
       "global": true,
-      "default_value": "Documentation"
+      "default_value": "Docs"
```

Comment:
Sorry if it was not clear syntax will only work after the new infra. For now you should only have one DOM selector (https://docsearch.algolia.com/docs/config-file/#setting-a-default-value):

```
"selectors": "#sidebar div.markerInNavPath:not(.selected)",
"default_value": "Docs"
```

Reply:
Got it! Fixed.

Reply:
Sorry for the confusion :D